zerberus/sfz.cpp

```diff
@@ -278,6 +286,18 @@ void ZInstrument::addRegion(SfzRegion& r)
             addZone(z);
       }
 
+//---------------------------------------------------------
+//   readFloat
+//---------------------------------------------------------
+
+static void readFloat(const QString& data, float& val)
+      {
+      bool ok;
+      double d = data.toFloat(&ok);
```

Comment:
A float is being assigned to a double here. Which is assigned to a float two lines later. I assume this function exists because you do not want to change the value of `delay` in case `opcode_data.toFloat()` fails. But you don't seem to be worried about `opcode_data.toUInt()` failing when setting the value of `offset`.

Speaking of which, `offset` is an `unsigned long`, so `opcode_data.toULong()` should probably be used when assigning its value.

Reply:
You are right! Thank you for review. It is not the last version, so I'll fix it.